src/actions/receive-profile.js

```diff
@@ -155,6 +157,7 @@ export function loadProfile(
       pathInZipFile: config.pathInZipFile,
       implementationFilter: config.implementationFilter,
       transformStacks: config.transformStacks,
+      dataSource: config.dataSource,
```

Comment:
I may be missing something, but I don't see where you're using this

Reply:
This must have been from an earlier implementation attempt. Nice catch, I'll remove it.